supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -82,3 +58,29 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
 
   override lazy val fileHashingFunction: Option[FileHashingFunction] = Option(FileHashingFunction(JesBackendFileHashing.getCrc32c))
 }
+
+object JesBackendLifecycleActorFactory {
```

Comment:
Probably a personal preference thing - I like to see the raison-d'etre of a class as close to the top of the file as possible and I find companion objects to mainly be helper-functions and the like